packages/web-pixels-extension/src/types/standard-events-V0/index.ts

```diff
@@ -0,0 +1,59 @@
+// Code generated by jtd-codegen for TypeScript v0.2.1
```

Comment:
These type are generated with the following command:

```bash
$ cd packages/web-pixels-extension/
$ jtd-codegen ./standard-events-schemas/StandardEventsV0.jtd.json --typescript-out ./src/types/standard-events-V0
```

The plan is to add the documentation on how to generate new event schema versions in the Pixel Manager private repo.